pkg/result/processors/diff.go

```diff
@@ -47,7 +49,7 @@ func (*Diff) Name() string {
 }
 
 func (p *Diff) Process(issues []result.Issue) ([]result.Issue, error) {
```

Comment:
Is it possible to cover the function with tests?

Reply:
I already thought about that: this is complex because it requires the creation of several git trees, and the tests will be very brittle and complex to maintain.
Also `Diff` is just a wrapper around `revgrep` and there are tests inside `revgrep`.